files/image_config/updategraph/updategraph

```diff
@@ -17,13 +10,6 @@ fi
 
 . /etc/sonic/updategraph.conf
 
-if [ "$enabled" = "reload_only" ]; then
```

Comment:
We can still keep the if statement, and echo an error message to stderr, and exit with non zero.